source/index.html.md.erb

```diff
@@ -49,7 +49,7 @@ These standards will enable us to maintain a consistent operating environment ac
 
 - [how to host a service](standards/hosting.html)
 - [managing DNS](standards/dns-hosting.html)
-- [monitoring your service](standards/monitoring.html)
+- [monitor your service](standards/monitoring.html)
```

Comment:
As with Logging, my keywords here are `monitoring` and `metrics`. Again, this is just a sample size of 1 so take accordingly.